ROADMAP.md

```diff
@@ -0,0 +1,43 @@
+# Four Keys 2021 Roadmap
+
+## Mission and Vision
+
+Four Keys mission:
+
+  Be the industry-standard reference implementation for the automated calculation of the DORA Four Keys metrics.  
+
+The vision for this is:
+
+* All discussion of the DORA Four Key metrics center around the key data entities of Changes, Deployments, and Incidents
```

Comment:
Grab language from PRD, make sure users understand that the only metrics this project will ever provide are the Four Key metrics.

Reply:
Added a little section for "Non-Goals"